### ISSUE TITLE
Feature bnf

### ISSUE THREAD

**Issue by simon-p-r** [rdking/chevrotain-ebnf#4]
Are there any plans to support bnf grammar files?

Thanks
Simon

**Comment by rdking** [comment]
No. But at the same time, that shouldn't be necessary. If you describe the BNF language in EBNF, then you'll get a BNF parser.
